.github/workflows/docker.yml

```diff
@@ -82,7 +83,8 @@ jobs:
       - name: Export digest
         id: digest
         if: github.event_name != 'pull_request'
-        run: | # zizmor: ignore[template-injection] docker/build-push-action output
+        run:
+          | # zizmor: ignore[template-injection] docker/build-push-action output
```

Comment:
existed before your PR, but have to ask. why are we ignoring this?

Reply:
that'd be one for @zerok

Reply:
Can also redirect that result through an env variable if that's now the recommended approach 🙂

Reply:
I'll fix it here!

Reply:
Oh right, it already did get fixed